ROADMAP.md

```diff
@@ -9,33 +9,36 @@ requests](https://github.com/PlasmaPy/plasmapy/pulls), and
 
 ## Features to be included in Version 0.1 release
 
+This is to be a preview/prototype release.
+
 - Plasma parameter calculations
```

Comment:
Perhaps add something like "Code and documentation infrastructure" to the top of the list?

Reply:
Why yes, we could. Sorry, I forgot.